proxy/http/HttpBodyFactory.cc

```diff
@@ -457,36 +437,29 @@ const char *
 HttpBodyFactory::determine_set_by_host(HttpTransact::State *context)
 {
   const char *set;
-  RawHashTable_Value v;
   int host_len = context->hh_info.host_len;
   char host_buffer[host_len + 1];
   strncpy(host_buffer, context->hh_info.request_host, host_len);
   host_buffer[host_len] = '\0';
-  if (table_of_sets->getValue((RawHashTable_Key)host_buffer, &v)) {
-    set = table_of_sets->getKeyFromBinding(table_of_sets->getCurrentBinding((RawHashTable_Key)host_buffer));
+  if (auto it = table_of_sets->find(host_buffer); it != table_of_sets->end()) {
```

Comment:
Since `table_of_sets` and `table_of_pages` are `unique_ptr`, it looks like we can't use range based for loops because they are things like `table_of_sets->begin()` instead of `table_of_sets.begin()`.

Reply:
You could use `.get` or even `*table_of_sets.get()`.